src/metrics.test.js

```diff
@@ -62,7 +62,7 @@ describe('Metrics', () => {
       const reporters = [new InMemoryReporter({ buffer: [] })];
 
       expect(() => new Metrics({ reporters, errback }))
-        .toThrow('errback must be a function');
+        .toThrow(TypeError);
```

Comment:
why not checking the exact message ?

Reply:
well, i can, but why? does the message itself represent a contract?